assets/prefabs/buildings/farm.prefab

```diff
@@ -0,0 +1,17 @@
+{
+    "GenericBuilding" : {
+     "name" : "Farm",
+     "templateNames" : ["MetalRenegades:farmTemplate"],
+     "zone" : "AGRICULTURE",
+     "minSize" : [9, 9],
```

Comment:
The indentation here is inconsistent.

Reply:
Indentation has been fixed, thanks for pointing this out!